ext/psych/lib/psych.rb

```diff
@@ -270,7 +270,7 @@ module Psych
   #
   def self.load yaml, legacy_filename = NOT_GIVEN, filename: nil, fallback: false, symbolize_names: false
     if legacy_filename != NOT_GIVEN
-      warn 'warning: Passing filename with the 2nd argument of Psych.load is deprecated. Use keyword argument like Psych.load(yaml, filename: ...) instead.'
+      warn 'Passing filename with the 2nd argument of Psych.load is deprecated. Use keyword argument like Psych.load(yaml, filename: ...) instead.', uplevel: 1
```

Comment:
`$VERBOSE` has been added to the showing condition to deprecation warnings in `ERB.new`.
The following is a similar change using keyword arguments.
https://github.com/ruby/ruby/blob/v2_6_0_preview2/lib/erb.rb#L808

If necessary, I'm going to add `$VERBOSE` condition to these deprecation warnings.

```diff
- warn 'Passing filename with the 2nd argument of Psych.load is deprecated. Use keyword argument like Psych.load(yaml, filename: ...) instead.', uplevel: 1
+ warn 'Passing filename with the 2nd argument of Psych.load is deprecated. Use keyword argument like Psych.load(yaml, filename: ...) instead.', uplevel: 1 if $VERBOSE
```